clang/docs/checkedc/3C/README.md

```diff
@@ -0,0 +1,64 @@
+# 3C: Semi-automated conversion of C code to Checked C
+
+The 3C (**C**hecked-**C**-**C**onvert) software facilitates conversion of existing C code to Checked C by automatically inferring as many Checked C annotations as possible and guiding the developer through the process of assigning the remaining ones.  3C aims to provide the first feasible way for organizations with large legacy C codebases (that they don't want to drop everything to rewrite in a better language) to comprehensively verify their code's spatial memory safety.
+
+## What 3C users should know about the development process
```

Comment:
Why is this part at the top of the README? Seems like this is not interesting for most readers, so it should go at the end.

Where should we talk about what the various options are that `3c` takes? 

And we should link the tutorial.

Reply:
> Why is this part at the top of the README? Seems like this is not interesting for most readers, so it should go at the end.

You are right.  There really isn't much to say right now other than the development process, but what there is should be moved to the top.  Done.

> Where should we talk about what the various options are that `3c` takes?

Done in `clang/tools/3c/README.md`.  If something is missing, please start a new thread there.

> And we should link the tutorial.

Done in `clang/tools/3c/README.md`; I think that's more appropriate.  If you disagree, please start a new thread.

Reply:
Seems good. Thanks.